src/opencolorio-1-fixes.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 13e2d64..47b5297 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,6 +9,12 @@ if(NOT DEFINED CMAKE_FIRST_RUN)
+     SET(CMAKE_FIRST_RUN 1 CACHE INTERNAL "")
+ endif()
+ 
++if (MSVC)
++    set(PATCH myptch)
++else()
++    set(PATCH patch)
++endif ()
++
+ ###############################################################################
+ ### GLOBAL ###
+ 
+@@ -171,12 +177,12 @@ else(USE_EXTERNAL_TINYXML)
+     endif()
+     ExternalProject_Add(tinyxml
+         URL ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.tar.gz
+-        PATCH_COMMAND patch -f -p1 < ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.patch
++        PATCH_COMMAND ${PATCH} --binary -f -p1 < ${CMAKE_SOURCE_DIR}/ext/tinyxml_${TINYXML_VERSION}.patch
+         BINARY_DIR ext/build/tinyxml
+         INSTALL_DIR ext/dist
+         CMAKE_ARGS ${TINYXML_CMAKE_ARGS}
+     )
+-    if(WIN32)
++    if(MSVC)
+         set(TINYXML_STATIC_LIBRARIES ${PROJECT_BINARY_DIR}/ext/dist/lib/tinyxml.lib)
+     else()
+         set(TINYXML_STATIC_LIBRARIES ${PROJECT_BINARY_DIR}/ext/dist/lib/libtinyxml.a)
+@@ -211,13 +217,13 @@ else(USE_EXTERNAL_YAML)
+     ExternalProject_Add(YAML_CPP_LOCAL
+         URL ${CMAKE_SOURCE_DIR}/ext/yaml-cpp-${YAML_CPP_VERSION}.tar.gz
+         BINARY_DIR ext/build/yaml-cpp
+-        PATCH_COMMAND patch -p1 < ${CMAKE_SOURCE_DIR}/ext/yaml-cpp-${YAML_CPP_VERSION}.patch
++        PATCH_COMMAND ${PATCH} --binary -p1 < ${CMAKE_SOURCE_DIR}/ext/yaml-cpp-${YAML_CPP_VERSION}.patch
+         INSTALL_DIR ext/dist
+         CMAKE_ARGS ${YAML_CPP_CMAKE_ARGS}
+     )
+     set(YAML_CPP_INCLUDE_DIRS ${PROJECT_BINARY_DIR}/ext/dist/include)
+     set(YAML_CPP_LIBRARY_DIRS ${PROJECT_BINARY_DIR}/ext/dist/lib)
+-    if(WIN32)
++    if(MSVC)
+         set(YAML_CPP_STATIC_DEBUG_LIBRARIES ${PROJECT_BINARY_DIR}/ext/dist/lib/libyaml-cppmdd.lib)
+         set(YAML_CPP_STATIC_OPTIMIZED_LIBRARIES ${PROJECT_BINARY_DIR}/ext/dist/lib/libyaml-cppmd.lib)
+     else()
+@@ -271,7 +277,7 @@ endif()
+ messageonce("SSE Optimizations: ${OCIO_USE_SSE}")
+ 
+ if(OCIO_USE_SSE)
+-    if(WIN32)
++    if(MSVC)
+         # SSE instructions are automatically compiled into 64-bit applications so enabling the option is redundant and
+         # actually produces an unknown option warning in Visual Studio.
+         if(NOT CMAKE_CL_64)
+diff --git a/docs/CMakeLists.txt b/docs/CMakeLists.txt
+index db42c2f..bd7dc02 100644
+--- a/docs/CMakeLists.txt
++++ b/docs/CMakeLists.txt
+@@ -55,7 +55,7 @@ set(SPHINX_VERSION 1.2b3)
+ ExternalProject_Add(Sphinx
+     DEPENDS setuptools docutils Jinja2 Pygments
+     URL ${CMAKE_SOURCE_DIR}/ext/Sphinx-${SPHINX_VERSION}.tar.gz
+-    PATCH_COMMAND patch -p1 < ${CMAKE_SOURCE_DIR}/ext/Sphinx-${SPHINX_VERSION}.patch
++    PATCH_COMMAND patch --binary -p1 < ${CMAKE_SOURCE_DIR}/ext/Sphinx-${SPHINX_VERSION}.patch
```

Comment:
Why is sphinx needed? As far as I know it is used to generate documentation, but MXE does not install documentation (and it seems to disabled by `-DOCIO_BUILD_DOCS=OFF`).